{{cookiecutter.project_repo_name}}/update_from_cc.py

```diff
@@ -0,0 +1,52 @@
+"""
+This script replays the cookiecutter repository creation with the original values.
+
+The intended use is to inject template changes into an existing repo.
+
+How-to
+======
+* Make sure that there are no files left uncommitted.
+* Check out the commit right after the repo creation by cookiecutter
+  and create a new branch that will hold the template modifications.
+* Clean the repo
+* Run this script from the parent directory
+* Commit all files that were created or modified
+* Checkout a new branch from master
+* Merge the modifications made to the original version
+* Fix the conflicts and merge
+
+.. code:: bash
+
+    git checkout -b orig_cc_update <SHA>
+    git clean -fxd
+    cd ..
+    python {{ cookiecutter.project_slug }}/update_from_cc.py
+    cd {{ cookiecutter.project_slug }}
+    git add * */.*
```

Comment:
I think there is a typo here, I used `git add * **/.*` but is `git add .` equivalent?

Reply:
On my end `git add .` did not add dot files.

Reply:
Good to know